database/migrations/postgres/1737404419146074_git_sync_status.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS git_sync_status
+(
```

Comment:
I think we should remove the eslVersion.

Our new approach is to split tables into "current" and "history".

However, for the git status, I don't think we want to store the history.
I would suggest to only store the current version in this table.